sql/migrations/jobsdb/000001_create_tables.down.tmpl

```diff
@@ -1,7 +1,7 @@
 -- Drop any existing datasets
 {{range .Datasets}}
-    DROP TABLE {{$.Prefix}}_jobs_{{.}};
-    DROP TABLE {{$.Prefix}}_job_status_{{.}};
+    DROP TABLE "{{$.Prefix}}_jobs_{{.}}";
```

Comment:
Is there an impact by modifying these? Will it cause them to run again? Do we have any kind of documentation on these migrations and how they work?

Reply:
No impact from some preliminary tests. Much of the jobsdb code has been modified to include `"`s as well, so I didn't think much into it.